app/containers/AskQuestion/renderTextarea.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TextEditor from 'components/TextEditor';
+
+import messages from './messages';
+
+function renderTextarea({ translations, disabled }) {
```

Comment:
Similar to above question. I assume we will have similar fields for other components. Can we put that somewhere where it can be reused?